hack/verify-vendor.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+WMCO_ROOT=$(pwd)
+
+# create a nice clean place to put our new vendor tree
```

Comment:
Please remove fluff like `nice clean place`. `temporary location` is sufficient. Applies everywhere.